web/frontend/src/index.tsx

```diff
@@ -224,38 +224,42 @@ const AppContainer = () => {
     };
 
     async function fetchData() {
+      const response = await fetch(ENDPOINT_PERSONAL_INFO, req);
+      let result;
       try {
-        const res = await fetch(ENDPOINT_PERSONAL_INFO, req);
-
-        if (res.status !== 200) {
-          const txt = await res.text();
-          throw new Error(`unexpected status: ${res.status} - ${txt}`);
+        if (response.status === 200) {
+          result = await response.json();
+        } else if (response.status === 401) {
+          result = {
+            isLoggedIn: false,
+            firstName: '',
+            lastName: '',
+            authorization: {},
+          };
+        } else {
+          const txt = await response.text();
+          throw new Error(`unexpected magic status: ${response.status} - ${txt}`);
```

Comment:
why magic? just regular HTTP 😁 
It doesn't even have to be unexpected: it could also be an error when dealing with a reverse proxy in front of the backend.
It looks to me more like an unhandled response, which practically should indeed translate to some form of error.

Reply:
agree on the comment on the error message, it should be just the status code and the reason

Reply:
Yeah, I removed it  😁